### PR TITLE
More test coverage, more tests

### PR DESCRIPTION
**Pull request by mchangrh**
coverage @ [f6f83fc](https://github.com/ajayyy/SponsorBlockServer/commit/f6f83fcbe43fb93b0f0391b6f392a4e9895cefa5) -> PR

% Stmts 81.05 -> 83.6
% Branch 66.78 -> 77.98
% Funcs 68.98 -> 77.72
% Lines 81.56 -> 84.45

### CODE REVIEW

**Comment by mchangrh** [comment]
# Changes
# Generic
- add `./coverage` directory to gitignore
# Mock Server
- restructured webhooks to all be under `/webhook` path
- move mocks to subdirectory
# NPM Scripts
- changed npm scripts `test:coverage` -> `cover`
- add npm scripts `cover:report`, `cover:genreport`
# routes
- explicitly send 0 when `getDaysSavedFormatted` fails
# utils
- getIP
  - return `127.0.0.1` for `getIP` when testing
  - remove `req.connection` since it was deprecated in node v16
# New Tests
- add `UserCounter` mock

**Comment by mchangrh** [comment]
There are still changes that I want to implement but it's at a good enough state for now, will continue webhook tests in #518

File: src/routes/verifyToken.ts [hunk 1]
```diff
@@ -65,20 +67,13 @@ export async function verifyTokenRequest(req: VerifyTokenRequest, res: Response)
...
-            const formData = new FormData();
-            formData.append("product_permalink", link);
-            formData.append("license_key", licenseKey);
-
-            const result = await axios.request({
-                url: "https://api.gumroad.com/v2/licenses/verify",
-                data: formData,
-                method: "POST",
-                headers: formData.getHeaders()
+            const result = await axios.post("https://api.gumroad.com/v2/licenses/verify", {
```
Comment (ajayyy) [inline src/routes/verifyToken.ts:70]: This is still form data?

Comment (mchangrh) [reply src/routes/verifyToken.ts:70]: https://help.gumroad.com/article/76-license-keys it can be done in parameters, doesn't need to be form data

**Pull request merged**